plugins/modules/digital_ocean_droplet.py

```diff
@@ -315,22 +330,23 @@ def main():
             oauth_token=dict(
                 aliases=['API_TOKEN'],
                 no_log=True,
-                fallback=(env_fallback, ['DO_API_TOKEN', 'DO_API_KEY', 'DO_OAUTH_TOKEN'])
+                fallback=(env_fallback, ['DO_API_TOKEN', 'DO_API_KEY', 'DO_OAUTH_TOKEN']),
+                required=True,
             ),
             name=dict(type='str'),
             size=dict(aliases=['size_id']),
             image=dict(aliases=['image_id']),
             region=dict(aliases=['region_id']),
-            ssh_keys=dict(type='list', no_log=False),
+            ssh_keys=dict(type='list', elements='str', no_log=False),
```

Comment:
This and the other `elements` additions should also have a changelog fragment.

Reply:
Cool, I think I have these all squared away now.